activerecord/lib/active_record/relation/calculations.rb

```diff
@@ -259,6 +259,9 @@ def execute_simple_calculation(operation, column_name, distinct) #:nodoc:
           column = aggregate_column(column_name)
 
           select_value = operation_over_aggregate_column(column, operation, distinct)
+          if operation == "sum" && distinct
```

Comment:
@rafaelfranca this does not seem to address the `average` and perhaps `calculate` use cases underlying the "calculations API" as well? It might be a surprise for others later with `sum` working as a one off like that. 

http://api.rubyonrails.org/classes/ActiveRecord/Calculations.html